dashboard/test/models/activity_test.rb

```diff
@@ -1,5 +1,6 @@
 require 'test_helper'
 
+require 'async_progress_handler'
```

Comment:
AsyncProgressHandler is dashboard specific since it ends up invoking Dashboard business logic to actually do the async operations, so I'm fine with following the rails "Don't include what you use" conventions.

By the way, we don't have any automated deployment for restarting the process_queues daemon when the code is updated, so this is technical dept in this area we are taking on.  After HOC I will either rip out out the async writes (if it turns out we didn't actually need it or want to migrate to DynamoDB or sharding) or fix this up.